_commands/toys/asciiquarium.md

```diff
@@ -3,22 +3,30 @@
 
 asciiquarium
 -------
-TODO: Add documentation for this command by submitting a pull request.
-<!-- one line explanation would go here -->
+`asciiquarium` is a script that doubles as a cool screensaver for the terminal.
 
-<!-- minimal example -->
-~~~ bash
-*short* example of command usage and output
-~~~
-
-<!--more-->
+To start `asciiquarium`, simply type the following:
 
-### Useful Options / Examples
+~~~bash
+$ asciiquarium
+~~~
 
-#### Example command
+Using ascii art, `asciiquarium` gives the terminal the feel of being out at sea
+with fish swimming around and the occasional fisherman. `asciiart`  is not built-in to your terminal. 
```

Comment:
"`asciiart` is not built-in to your terminal." Is asciiart a dependency of asciiquarium? If so, could we include some steps to install it?